interface.js

```diff
@@ -0,0 +1,36 @@
+view.addEventListener('click', function(){
```

Comment:
This is cool. I didn't realise you could reference the ids as variable names directly!